index.html

```diff
@@ -38,6 +39,203 @@ <h1 class="text1">
             <li><a href="https://angellist.com"><img src="images/angellist.png" alt="angellist"></a></li>
         </ul>   
     </div>
+</section> -->
+
+<section class="grid-container">
+    <div class="snapshot-image">
+        <img src="images/Snapshoot PortfolioNew.png" alt="">
+    </div>
+    <div class="tonicmain">
+        <p>Tonic</p>
```

Comment:
[OPTIONAL]

- [ ] Kindly ensure that the project title (e.g., "Tonic") uses a header tag hx( i.e I will recommend you use the `<h2>` tag) 👍🏽